pkg/component/component.go

```diff
@@ -214,7 +218,7 @@ func GetComponentSource(client *occlient.Client, componentName string, applicati
 		sourceType = "git"
 		sourcePath = bc.Spec.Source.Git.URI
 	case buildv1.BuildSourceBinary:
-		sourceType = "local"
+		sourceType = bc.ObjectMeta.Annotations[componentSourceTypeAnnotation]
```

Comment:
Please update comment for this function (`GetComponentSource`). It says that component source type can be "git" or "local"